### PR TITLE
fix: Tradingcompetition mobox

### PR DESCRIPTION
**Pull request by memoyil**

### CODE REVIEW

File: src/views/TradingCompetition/components/ClaimModal/index.tsx [hunk 1]
```diff
@@ -73,19 +64,23 @@ const ClaimModal: React.FC<CompetitionProps> = ({ onDismiss, onClaimSuccess, use
...
-        <Heading mt="16px" scale="md" mb={canClaimNFT ? '16px' : '0px'}>
-          {santosReward.toFixed(2)} SANTOS
+          {moboxReward.toFixed(2)} MBOX
         </Heading>
         {/* NFT */}
         {canClaimNFT ? (
           <Flex alignItems="center" flexDirection="column" width="100%">
             <ImageWrapper>
-              <Image src={NftBunnies} width={128} height={128} />
+              <Image src={MboxAllBunnies} width={128} height={128} />
```
Comment (memoyil) [inline src/views/TradingCompetition/components/ClaimModal/index.tsx:73]: Is the image correct or do we need another?

File: src/views/TradingCompetition/MoboxCompetition.tsx [hunk 2]
```diff
@@ -160,48 +161,50 @@ const MoboxCompetition = () => {
...
 
-  //   useEffect(() => {
-  //     const fetchGlobalLeaderboardStats = async () => {
-  //       const res = await fetch(`${profileApiUrl}/api/leaderboard/global`)
-  //       const data = await res.json()
-  //       setGlobalLeaderboardInformation(data)
-  //     }
+  useEffect(() => {
+    const fetchGlobalLeaderboardStats = async () => {
+      const res = await fetch(`${profileApiUrl}/api/leaderboard/global`)
```
Comment (chef-jojo) [inline src/views/TradingCompetition/MoboxCompetition.tsx:174]: `?competitionId=3`

File: src/views/TradingCompetition/MoboxCompetition.tsx [hunk 3]
```diff
@@ -160,48 +161,50 @@ const MoboxCompetition = () => {
...
-  //         const res = await fetch(`${profileApiUrl}/api/leaderboard/team/${teamId}`)
-  //         const data = await res.json()
-  //         callBack(data)
-  //       } catch (e) {
-  //         console.error(e)
-  //       }
-  //     }
+    const fetchTeamsLeaderboardStats = async (teamId: number, callBack: (data: any) => void) => {
+      try {
+        const res = await fetch(`${profileApiUrl}/api/leaderboard/team/${teamId}`)
```
Comment (chef-jojo) [inline src/views/TradingCompetition/MoboxCompetition.tsx:181]:
```suggestion
        const res = await fetch(`${profileApiUrl}/api/leaderboard/3/team/${teamId}`)
```

File: src/views/TradingCompetition/components/ClaimModal/index.tsx (hunk 1 above)
Comment (Chef-Cheems) [reply src/views/TradingCompetition/components/ClaimModal/index.tsx:73]: For not this is ok. We'll prepare proper image by the claim phase

**Review by Chef-Cheems — APPROVED** [review]

**Pull request merged**